packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -113,6 +108,11 @@ export class CoinbaseWalletSDK {
 
     this._relayEventManager = new WalletSDKRelayEventManager();
 
+    const isMobile = isMobileWeb();
+    const uiConstructor =
+      options.uiConstructor ||
+      (opts => new (isMobile ? MobileRelayUI : WalletLinkRelayUI)(opts));
```

Comment:
while this works its kind of hard to read

```ts
(opts => new (isMobile ? MobileRelayUI : WalletLinkRelayUI)(opts));
```

the more verbose version is a bit easier to read

```ts
(opts => isMobile ? new MobileRelayUI(opts) : new WalletLinkRelayUI(opts);
```

Reply:
thanks for the feedback. i wasn't aware what js convention on this was, so thought this version was easier to read. will make change.